src/components/pages/StudentsPage/Conversation.tsx

```diff
@@ -11,16 +11,25 @@ export default function Conversation({ socket, chat, setChat }) {
   const lastMessage = useRef(null);
   useEffect(() => {
     if (socket) {
-      socket.on('chat message', ({ character, message }) => {
+      socket.on('student sent message', ({ character, message }) => {
         setChat((chat) => ({
           ...chat,
           conversation: [...chat.conversation, ['peer', character, message]],
         }));
       });
+      socket.on('teacher sent message', ({ message }) => {
+        setChat((chat) => ({
+          ...chat,
+          conversation: [...chat.conversation, ['teacher', 'TEACHER', message]],
+        }));
+      });
```

Comment:
Nit: This works for now, but I personally would have liked to see the `chat message` reused and/or refactored to be reusable.

What happens down the road when we want to add multiple users that could potentially target a any given chat?  The pattern being established here forces us to create new (mostly duplicated code) to handle each exception... rather than generically handling a chat message from any user.